sources/gerritDB/index.js

```diff
@@ -22,13 +24,18 @@ var gerritDB = (function () {
         "DRAFT": "d"
     };
 
-    function doQuery (query, callback) {
+    function doQuery (fn, query, callback) {
         var connection = Connection.connect();
+        var debugStr = "[" + fn + "] ";
+
+        debug(debugStr + query);
```

Comment:
no need. it's a special debug line which is only activated when DEBUG=* is written on CLI.